modules/oadp-installing-dpa.adoc

```diff
@@ -293,31 +303,33 @@ spec:
         - gcp <2>
         - csi <3>
         - openshift <4>
+      resourceTimeout: 10m <5>
     restic:
-      enable: true <5>
+      enable: true <6>
       podConfig:
-        nodeSelector: <node selector> <6>
+        nodeSelector: <node selector> <7>
   backupLocations:
     - velero:
-        provider: {provider} <7>
+        provider: {provider} <8>
         default: true
         credential:
           key: cloud
-          name: <default_secret> <8>
+          name: <default_secret> <9>
         objectStorage:
-          bucket: <bucket_name> <9>
-          prefix: <prefix> <10>
+          bucket: <bucket_name> <10>
+          prefix: <prefix> <11>
 ----
 <1> The `kubevirt` plugin is mandatory for {VirtProductName}.
 <2> Specify the plugin for the backup provider, for example, `gcp`, if it exists.
 <3> The `csi` plugin is mandatory for backing up PVs with CSI snapshots. The `csi` plugin uses the link:https://{velero-domain}/docs/main/csi/[Velero CSI beta snapshot APIs]. You do not need to configure a snapshot location.
 <4> The `openshift` plugin is mandatory.
-<5> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that each worker node has `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
-<6> Specify the node selector to be supplied to Restic podSpec
-<7> Specify the backup provider.
-<8> If you use a default plugin for the backup provider, you must specify the correct default name for the `Secret`, for example, `cloud-credentials-gcp`. If you specify a custom name, the custom name is used for the backup location. If you do not specify a `Secret` name, the default name is used.
-<9> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
-<10> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
+<5> Set how long to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. Default is 10m.
+<6> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that each worker node has `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
+<7> Specify the node selector to be supplied to Restic podSpec
+<8> Specify the backup provider.
+<9> If you use a default plugin for the backup provider, you must specify the correct default name for the `Secret`, for example, `cloud-credentials-gcp`. If you specify a custom name, the custom name is used for the backup location. If you do not specify a `Secret` name, the default name is used.
```

Comment:
Transitive verb formatting. Could we try something like:

Specify the correct default name for the `Secret`, for example, `cloud-credentials-gcp`, if you use a default plugin for the backup provider. If specifying a custom name, then the custom name is used for the backup location. If you do not specify a `Secret` name, the default name is used.